src/discordTools/SetupSettingsMenu.js

```diff
@@ -75,6 +75,17 @@ async function setupGeneralSettings(client, guildId, channel) {
             Path.join(__dirname, '..', 'resources/images/settings_logo.png'))]
     });
 
+    await client.messageSend(channel, {
+        embeds: [DiscordEmbeds.getEmbed({
+            color: Constants.COLOR_SETTINGS,
+            title: client.intlGet(guildId, 'commandsVcGenderDesc'),
+            thumbnail: `attachment://settings_logo.png`
+        })],
+        components: [DiscordSelectMenus.getVoiceGenderSelectMenu(guildId, instance.generalSettings.gender)],
```

Comment:
change generalSettings gender to voiceGender, so we know why we have a gender setting